extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -230,6 +231,15 @@ public void pause() {
         }
     }
 
+    @Override
+    public void pause(String identity) {
+        try {
+            scheduler.pauseJob(new JobKey(identity, Scheduler.class.getName()));
+        } catch (SchedulerException e) {
+            LOGGER.warn("Unable to pause job", e);
```

Comment:
I wonder if we shouldn't change the strategy and wrap the `SchedulerException` in a `RuntimeException` and rethrow from all pause/resume methods...

Reply:
I removed log and throw now RuntimeException with wrapped SchedulerException inside